content/body/tooltip.php

```diff
@@ -97,12 +97,12 @@
 </p>
 
 <ul>
-  <li>For <a href="https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/H28">providing definitions to abbreviations</a> using the <code>&lt;abbr&gt;</code> tag (However, <a href="https://twitter.com/stevefaulkner">Steve Faulkner</a> suggests <a href="https://www.tpgi.com/short-note-the-abbreviation-appreciation-society/">other methods for expanding abbreviations in a more user friendly way</a>.</li>
+  <li>For <a href="https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/H28">providing definitions to abbreviations</a> using the <code>&lt;abbr&gt;</code> tag (However, <a href="https://twitter.com/stevefaulkner">Steve Faulkner</a> suggests <a href="https://www.tpgi.com/short-note-the-abbreviation-appreciation-society/">other methods for expanding abbreviations in a more user-friendly way</a>.</li>
   <li>For <a href="https://dequeuniversity.com/tips/provide-iframe-titles">providing titles to iFrames</a> (which has nothing to do with its tooltip functionality).</li>
 </ul> 
 
 <p>
-  A really good round up of how the <code>title</code> attribute works, its history, and where it is appropriate to
+  A really good round-up of how <code>title</code> attribute works, its history, and where it is appropriate to
```

Comment:
I still think the "the" word is needed:

```suggestion
  A really good round-up of how the <code>title</code> attribute works, its history, and where it is appropriate to
```